AzureCalling/Utilities/AudioSessionManager.swift

```diff
@@ -46,4 +46,15 @@ class AudioSessionManager {
             }
         }
     }
+    public static func selectedAudioDeviceButtonIcon() -> UIImage {
```

Comment:
Not sure if it's a good idea to read the icon image from AudioSessionManager. 
Maybe extend the `AudioDeviceType` to be an enum with associated value of iconImage. Rather to put this function here.

Reply:
@AmyL219 Thanks for the suggestion. It's a good idea which also helps AudioSessionManager maintain its core and cleans up the code. Suggestions now implemented and PR updated.